src/Docker/createpackage.msbuild

```diff
@@ -1,6 +1,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="CreatePackage"  ToolsVersion="4.0">
 	<PropertyGroup>
 		<GXDeployFileProject>$([System.IO.Path]::GetFullPath('$(DeployFullPath)\..\..\..\..\web\$(ProjectName).gxdproj'))</GXDeployFileProject>
+		<RedissonLibs>$([System.IO.Path]::GetFullPath('$(DeployFullPath)\..\..\context\tomcat\lib'))</RedissonLibs>
```

Comment:
Please, think of use / as path separator. \ is not cross-platform (we will have to change them to be used at GeneXus Next Edition.